ArcadiaParties-SPA/src/app/user/user-info/user-info.component.html

```diff
@@ -0,0 +1,19 @@
+<div class="flex-container">
```

Comment:
You don't need to have additional flex container, because your can add styles to your host component.

Reply:
It fixed

Reply:
No, it is not. `flex-container` is still in your template.

Reply:
Still here.